### PR TITLE
Chore: Add Fog, Anoop and Daniel as codeowners

### PR DESCRIPTION
**Pull request by anoop2811**
### Description of your changes

copilot:all

Fixes #

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+*                                  @barnettZQG @wonderflow @leejanee @Somefive @jefree-cat @FogDong @wangyikewxgm @chivalryq @anoop2811 @dhiguero
+design/                            @barnettZQG @leejanee @wonderflow @Somefive @jefree-cat @FogDong @anoop2811 @dhiguero
 
 # Owner of Core Controllers
-pkg/controller/core.oam.dev       @Somefive @FogDong @barnettZQG @wonderflow @wangyikewxgm @chivalryq
+pkg/controller/core.oam.dev       @Somefive @FogDong @barnettZQG @wonderflow @wangyikewxgm @chivalryq @anoop2811 @dhiguero
 
 # Owner of Standard Controllers
-pkg/controller/standard.oam.dev   @wangyikewxgm @barnettZQG @wonderflow @Somefive
+pkg/controller/standard.oam.dev   @wangyikewxgm @barnettZQG @wonderflow @Somefive @anoop2811 @dhiguero
```
Comment (FogDong) [inline .github/CODEOWNERS:10]: Could you please also add me as the owner?

Comment (FogDong) [inline .github/CODEOWNERS:10]:
```suggestion
pkg/controller/standard.oam.dev   @wangyikewxgm @barnettZQG @wonderflow @Somefive @anoop2811 @dhiguero @FogDong
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+pkg/cue                            @leejanee @FogDong @Somefive @anoop2811 @dhiguero
+pkg/stdlib                         @leejanee @FogDong @Somefive @anoop2811 @dhiguero
 
 # Owner of Workflow
-pkg/workflow                       @leejanee @FogDong @Somefive @wangyikewxgm @chivalryq
+pkg/workflow                       @leejanee @FogDong @Somefive @wangyikewxgm @chivalryq @anoop2811 @dhiguero
 
 # Owner of vela templates
-vela-templates/                     @Somefive @barnettZQG @wonderflow @FogDong @wangyikewxgm @chivalryq
+vela-templates/                     @Somefive @barnettZQG @wonderflow @FogDong @wangyikewxgm @chivalryq @anoop2811 @dhiguero
```
Comment (FogDong) [inline .github/CODEOWNERS:20]:
```suggestion
vela-templates/                     @Somefive @barnettZQG @wonderflow @FogDong @wangyikewxgm @chivalryq @anoop2811 @dhiguero @FogDong 
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 
 # Owner of vela CLI
-references/cli/                     @Somefive @zzxwill @StevenLeiZhang @charlie0129 @wangyikewxgm @chivalryq
+references/cli/                     @Somefive @zzxwill @StevenLeiZhang @charlie0129 @wangyikewxgm @chivalryq @anoop2811 @dhiguero
```
Comment (FogDong) [inline .github/CODEOWNERS:23]:
```suggestion
references/cli/                     @Somefive @zzxwill @StevenLeiZhang @charlie0129 @wangyikewxgm @chivalryq @anoop2811 @dhiguero @FogDong 
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,35 +1,35 @@
...
 
 # Owner of vela addon framework
-pkg/addon/                         @wangyikewxgm @wonderflow @charlie0129
+pkg/addon/                         @wangyikewxgm @wonderflow @charlie0129 @anoop2811 @dhiguero
```
Comment (FogDong) [inline .github/CODEOWNERS:26]:
```suggestion
pkg/addon/                         @wangyikewxgm @wonderflow @charlie0129 @anoop2811 @dhiguero @FogDong 
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (anoop2811) [reply .github/CODEOWNERS:10]: sure, will add you as well

**Review by FogDong — APPROVED** [review]

**Pull request merged**